react-api-client/src/camera/useCameraImageSettings.ts

```diff
@@ -0,0 +1,68 @@
+import { useMutation, useQueryClient } from 'react-query'
+
+import { getCameraImageSettings } from '@opentrons/api-client'
+import { OT_SYSTEM_CAMERA } from '@opentrons/shared-data'
+
+import { useHost } from '../api'
+
+import type { AxiosError } from 'axios'
+import type {
+  UseMutateFunction,
+  UseMutationOptions,
+  UseMutationResult,
+} from 'react-query'
+import type {
+  CameraImageSettings,
+  CameraImageSettingsResponse,
+  ErrorResponse,
+} from '@opentrons/api-client'
+
+export type UseCameraImageSettingsMutationResult = UseMutationResult<
+  CameraImageSettingsResponse,
+  AxiosError<ErrorResponse>,
+  CameraImageSettings
+> & {
+  updateCameraImageSettings: UseMutateFunction<
+    CameraImageSettingsResponse,
+    AxiosError<ErrorResponse>,
+    CameraImageSettings
+  >
+}
+
+export function useCreateCameraImageSettings(
+  options: UseMutationOptions<
+    CameraImageSettingsResponse,
+    AxiosError<ErrorResponse>,
+    CameraImageSettings
+  > = {}
+): UseCameraImageSettingsMutationResult {
+  const host = useHost()
+  const queryClient = useQueryClient()
+
+  const mutation = useMutation<
+    CameraImageSettingsResponse,
+    AxiosError<ErrorResponse>,
+    CameraImageSettings
+  >(
+    [host, 'camera', 'cameraSettings', OT_SYSTEM_CAMERA],
```

Comment:
```suggestion
const cameraId = OT_SYSTEM_CAMERA
...
    [host, 'camera', 'cameraSettings', cameraId],
```
Somewhat nitty, but this will help reduce bugs for future devs, as it's pretty easy to glaze over the query key line.